crates/weaver_forge/src/lib.rs

```diff
@@ -421,6 +422,23 @@ fn flatten(value: Value) -> Result<Value, minijinja::Error> {
     Ok(Value::from(result))
 }
 
+// Helper function to take an "id" and split it by '.' into namespaces.
+fn split_id(value: Value) -> Result<Vec<Value>, minijinja::Error> {
+    match value.as_str() {
+        Some(id) => {
+            let values: Vec<Value> = id
+                .split('.')
+                .map(|s| Value::from_safe_string(s.to_owned()))
+                .collect();
+            Ok(values)
```

Comment:
```suggestion
            Ok(Value::from(values))
```

Reply:
FYI - merged this suggestion and it breaks everything.

Reply:
Ok